recipes/_master_base_config.rb

```diff
@@ -33,12 +33,18 @@
   creates dev_path
 end
 
+reboot 'ec2_nvme_reboot' do
+  action :reboot_now
+  reason 'Reboot so the NVMe driver can discover the newly attached EBS volume.'
+  subscribes :reboot_now, "execute[attach_volume]", :immediately
+end
+
 # wait for the drive to attach, before making a filesystem
 ruby_block "sleeping_for_volume" do
   block do
     wait_for_block_dev(dev_path)
   end
-  subscribes :run, "execute[attach_volume]", :immediately
+  subscribes :run, "reboot[ec2_nvme_reboot]", :immediately
```

Comment:
why would we do this all this time?  Seems better to only do it with NVMe volumes.  Even then, not sure that I love the reboot here.  Shouldn't the volume be attached when the instance launches with the cloudformation template?